File: server-ASP.NET/Domain/Message.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    // public enum Type { text, audio, image, video }

    public class Message
    {
        public int Id { get; set; }

        public string Content { get; set; }

        [DataType(DataType.DateTime)]
        public string Created { get; set; }

        public bool Sent { get; set; }
    }
}

File: client-react/src/Components/Message/Message.js
import "./Message.css"

function Message(props) {
    const {message} = props;

    /*let msg;
    if(message.type==="text"){
        msg=message.content;
    }
    else if(message.type==="audio")
    {
        msg= <audio controls>
                <source src={message.content} type="video/webm"/>
            </audio>;
    }
    else if(message.type==="image")
    {
        msg= <img src={message.content} className="img-fluid" alt="image..."/>;
    }
    else if(message.type==="video")
    {
       msg= <video width="640" height="360" controls>
            <source src={message.content} type="video/mp4" />
            <source src={message.content} type="video/ogg" />
            <source src={message.content} type="video/webm" />
        </video>
    }*/

    if (message?.sent) {
        return (
            <p className={"chat_message sender"}>
                {message?.content}
                <span className="time_stamp ">
                    {message?.created}
                </span>
            </p>
        );
    } else {
        return (
            <p className={"chat_message"}>
                {message?.content}
                <span className="time_stamp ">
                    {message?.created}
                </span>
            </p>
        );
    }

}
export default Message;